src/linker/utilities/docker_utils.py

```diff
@@ -68,6 +68,8 @@ def _run_container(
         },
         str(results_dir): {"bind": "/results", "mode": "rw"},
         str(diagnostics_dir): {"bind": "/diagnostics", "mode": "rw"},
+        # Untested
+        "/tmp": {"bind": "/tmp", "mode": "rw"},
```

Comment:
singularity just binds "/tmp":"/tmp" generically because R requires it, so I thought it would be fine to do the same for docker/ I'm not in a position to test this out though. Don't know if it's best to keep or remove it given that we're going to drop docker support for a while.

Reply:
Yeah it's fine - we've agreed to just wholesale delete docker for now anyway